NGitLab/Models/ReleaseInfo.cs

```diff
@@ -3,7 +3,7 @@
 namespace NGitLab.Models
 {
     [DataContract]
-    public class RealeaseInfo
+    public class ReleaseInfo
```

Comment:
Snake eyes!

Reply:
Can't resist :) sooner or later it's going to be a breaking change.